UI/GameSettingsScreen.cpp

```diff
@@ -1103,6 +1103,7 @@ void GameSettingsScreen::CreateViews() {
 		vrSettings->Add(new ItemHeader(vr->T("VR camera")));
 		vrSettings->Add(new PopupSliderChoiceFloat(&g_Config.fCanvasDistance, 1.0f, 15.0f, vr->T("Distance to 2D menus and scenes"), 1.0f, screenManager(), ""));
 		vrSettings->Add(new PopupSliderChoiceFloat(&g_Config.fFieldOfViewPercentage, 100.0f, 200.0f, vr->T("Field of view scale"), 10.0f, screenManager(), vr->T("% of native FoV")));
+		vrSettings->Add(new PopupSliderChoiceFloat(&g_Config.fHeadUpDidplayScale, 0.2f, 1.0f, vr->T("Head up display scale"), 0.1f, screenManager(), ""));
```

Comment:
Note: en_US usage is usually Heads-up display, while en_GB and I believe other usage is typically Head-up.

-[Unknown]